openedx/core/djangoapps/coursegraph/management/commands/tests/test_dump_to_neo4j.py

```diff
@@ -297,8 +297,8 @@ def _extract_relationship_pairs(relationships, relationship_type):
               constituent nodes.
         """
         relationship_pairs = [
-            tuple([node["location"] for node in rel.nodes()])  # lint-amnesty, pylint: disable=consider-using-generator
-            for rel in relationships if rel.type() == relationship_type
+            (rel.start_node["location"], rel.end_node["location"])
+            for rel in relationships if type(rel).__name__ == relationship_type
```

Comment:
The `Node` API changed a bit, but the mapping from the old API to the new was fairly straightforward.